style/style.css

```diff
@@ -50,3 +50,71 @@ form button {
   display: flex;
   margin: 5px 0 0 auto;
 }
+
+form input {
+  margin-top: 10px;
+  font-size: 15px;
+}
+
+.nav {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  float: right;
+}
+
+.nav-item {
+  float: left;
+}
+
+.nav-item a {
+  display: block;
+  color: black;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+  border: none;
+}
+
+nav {
+  display: inline;
+}
```

Comment:
- [ ] [REQUIRED] You should place a `vertical line` between each navigation item.

![image](https://user-images.githubusercontent.com/63915024/182651652-debd203c-2c1b-4229-9139-80147138d0be.png)